src/python/alog/alog.py

```diff
@@ -177,6 +177,9 @@ def format(self, record):
             else:
                 log_record['message'] = str(record_args)
 
+        if hasattr(record, "additional_ctxt_metadata"):
```

Comment:
If we're going to have a "special" string for this, I think it should be (a) a constant, and (b) a `__dunder__` name

Reply:
Yes, certainly. Will update the PR